README.md

```diff
@@ -42,4 +40,30 @@ The JVM will then listen for a debugger to be attached on port 8189.  The JVM wi
 the debugger to be attached before starting Echo; the relevant JVM arguments can be seen and
 modified as needed in `build.gradle`.
 
-[//]: # "Only here to retrigger the echo build"
+## Configuring
```

Comment:
@robzienert what do you think about this doc (it's incomplete for now, obv).
Wanted your opinion that this is right direction before I flesh it out

Reply:
this is good. We’ll want to have more in-depth docs about setup and operations in spinnaker.io - but I do like a crash course in the README... we should probably do the same for other services.